mac_build/notarize.js

```diff
@@ -1,18 +1,26 @@
 /* eslint-disable @typescript-eslint/no-require-imports */
-
-const result = require("dotenv").config({ path: "./mac_build/.env", debug: true })
 const { notarize } = require("electron-notarize")
 
-if (result.error) {
-  throw result.error
+function isEnvFilePresent() {
+  const result = require("dotenv").config({ path: "./mac_build/.env", debug: true })
+  if (result.error?.code === "ENOENT") {
+    console.log("  • ⚠️ No mac_build/.env detected. Skipping notarization step.")
+    return false
+  }
+
+  return true
 }
 
 exports.default = async function notarizing(context) {
-  if (!result || process.env.SKIP_NOTARIZE === "true") {
-    console.log("  • No mac_build/.env or SKIP_NOTARIZE=true detected. Skipping notarization step.")
+  if (!isEnvFilePresent()) {
```

Comment:
Is there a way to just skip notarization by detecting nosign in `npm run package:mac:nosign`?